version/Core/Public/API/ARK/PrimalStructure.h

```diff
@@ -661,7 +661,7 @@ struct APrimalStructureBed : APrimalStructure
 	void Destroyed() { NativeCall<void>(this, "APrimalStructureBed.Destroyed"); }
 	void BeginPlay() { NativeCall<void>(this, "APrimalStructureBed.BeginPlay"); }
 	FSpawnPointInfo* GetSpawnPointInfo(FSpawnPointInfo* result) { return NativeCall<FSpawnPointInfo*, FSpawnPointInfo*>(this, "APrimalStructureBed.GetSpawnPointInfo", result); }
-	static APrimalStructure* FindBedWithID(UWorld* forWorld, int theBedID) { return NativeCall<APrimalStructure*, UWorld*, int>(nullptr, "APrimalStructureBed.FindBedWithID", forWorld, theBedID); }
+	static AActor* FindBedWithID(UWorld* forWorld, int theBedID) { return NativeCall<AActor*, UWorld*, int>(nullptr, "APrimalStructureBed.FindBedWithID", forWorld, theBedID); }
```

Comment:
IDA shows this as `AActor*` now, and looking at some disassembled code
```
      v3 = this->GetWorld(this);
      BedWithID = APrimalStructureBed::FindBedWithID(v3, fromSpawnPointID);
      v4 = this->GetWorld(this);
      v14 = APrimalStructureBed::FindBedWithID(v4, spawnPointID);
      if ( BedWithID )
      {
        if ( v14
          && !BedWithID->IsPrimalDino(BedWithID)
          && !v14->IsPrimalDino(v14)
```
we see the hint `BedWithID->IsPrimalDino` that indicates that beds/spawn points are not always `APrimalStructure*`